src/Engines/TestWare.Engines.Selenium/Factory/ChromeDriver.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium.Chrome;
+
+namespace TestWare.Engines.Selenium.Factory;
+
+public class ChromeDriver : OpenQA.Selenium.Chrome.ChromeDriver, IBrowserDriver
+{
+    public ChromeDriver()

```

Comment:
Default constructor doesn't need to be specified
https://www.geeksforgeeks.org/c-sharp-default-constructor/

Reply:
deleted